blob/fileblob/fileblob.go

```diff
@@ -74,15 +74,26 @@ func init() {
 }
 
 // Options sets options for constructing a *blob.Bucket backed by fileblob.
-type Options struct{}
+type Options struct {
+	URLSigner URLSigner
+}
 
 type bucket struct {
-	dir string
+	dir       string
+	urlSigner URLSigner
+}
+
+func (b *bucket) path(key string) (string, error) {
```

Comment:
nit: Let's move this method below the constructor functions so it's grouped with the other methods.